### ISSUE TITLE
advanced search , date from and date to are invalid dates

### ISSUE THREAD

**Issue by cbruyndoncx** [newburns/osCommerce-234-bootstrap-wADDONS#44]
selected from and to dates via the calendar widget, 
tried searching without keywords entered - failed - OK
added keyword and searched, but gives invalid dates error for both in 2 error messages

**Comment by cbruyndoncx** [comment]
further info, 
upon selection of the date, it shows preformatted mm/dd/yyyy but upon submission afterwards it shows without the delimiters as mmddyyyy

**Comment by newburns** [comment]
Seems to be an issue with Security Pro addon.
Edit catalog/includes/application_top.php and remove /* ** Security Pro FWR ** */ coding and retest.
Does it resolve?

**Comment by cbruyndoncx** [comment]
yes, disabling the cleanse of PHP SELF solves it, but that is a workaround.
I'm not familiar with that contribution, any bright idea to keep the date filter functionality and security pro ?

**Comment by tmccaff** [comment]
I will look into this.

**Comment by newburns** [comment]
@tmccaff @cbruyndoncx 
Was there any resolution or suggestions?
It appears that this addon may need to be disabled completely if there are no ideas
